doc/docs/master/concepts/advanced-concepts/deferred_processing.md

```diff
@@ -284,6 +284,40 @@ When more than one is specified, a branch repoint will be triggered when any of
 the conditions is met. To guarantee that they all must be met, add
 --trigger-all.
 
+## Embed Triggers in Pipelines
+
+Triggers can also be specified in the pipeline spec and automatically created
+when the pipeline is created. For example, this is the edges pipeline from our
+our OpenCV demo modified to only trigger when there is a MB of new images:
+
+```
+{
+  "pipeline": {
+    "name": "edges"
+  },
+  "description": "A pipeline that performs image edge detection by using the OpenCV library.",
+  "input": {
+    "pfs": {
+      "glob": "/*",
+      "repo": "images",
+      "trigger": {
+          "size": "1MB"
+      }
+    }
+  },
+  "transform": {
+    "cmd": [ "python3", "/edges.py" ],
+    "image": "pachyderm/opencv"
+  }
+}
+```
+
+When you create this pipeline Pachyderm will also create a branch in the input
```

Comment:
separate the introductory phrase with a comma:

When you create this pipeline,  Pachyderm will also create a branch in the input

Reply:
```suggestion
When you create this pipeline, Pachyderm will also create a branch in the input
```